examples/albert/run_trainer.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python
+
+import logging
+import os
+from dataclasses import dataclass, field, asdict
+from pathlib import Path
+from typing import Optional, Dict, Any
+import uuid
+
+from datasets import load_from_disk
+import transformers
+from torch.utils.data import DataLoader
+from transformers import (set_seed, HfArgumentParser, TrainingArguments,
+                          DataCollatorForLanguageModeling, AlbertTokenizerFast, AlbertConfig, AlbertForPreTraining)
+from transformers.optimization import get_linear_schedule_with_warmup
+from transformers.trainer_utils import is_main_process
+from transformers.trainer import Trainer
+from torch_optimizer import Lamb
+import torch
+
+import hivemind
+
+
+logger = logging.getLogger(__name__)
+LRSchedulerBase = getattr(torch.optim.lr_scheduler, '_LRScheduler', None)
+
+
+@dataclass
+class CollaborationArguments:
+    """ define how peers interact with each other while training"""
+
+    # primary parameters
+    initial_peers: str  # one or more peers (comma-separated) that will welcome you into the collaboration
```

Comment:
If we define this as a list field [similarly to examples](https://github.com/huggingface/transformers/blob/master/tests/test_hf_argparser.py#L75), we will avoid parsing this string by hand